examples/src/bin/tlsserver-mio.rs

```diff
@@ -444,8 +444,9 @@ Options:
                         CRLFILE. May be used multiple times.
     --require-auth      Send a fatal alert if the client does not complete client
                         authentication.
-    --resumption        Support session resumption.
-    --tickets           Support tickets.
+    --no-resumption     Disable stateful session resumption.
+    --tickets           Support tickets (stateless resumption).
+    --max-early-data BYTES   Support receiving BYTES many bytes with 0RTT
```

Comment:
Because these flags are reproduced in the README you'll need to do an update dance to fix [this CI job that's failing](https://github.com/rustls/rustls/actions/runs/10147456601/job/28059069867?pr=2059).

Try running `./admin/pull-readme` in the commit that changes these flags, and then add the updated `README.md` content to the commit.

Reply:
It actually will be `./admin/pull-usage` for this one.

Reply:
Oops, you're right. Thanks!

Reply:
Thanks for the tip. The README is updated with [f1c774d](https://github.com/rustls/rustls/pull/2059/commits/f1c774d34bce1d7d5ca1a8af64569b42160394fc)